assembl/tests/fixtures/idea_message_columns.py

```diff
@@ -29,6 +29,18 @@ def fin():
     return column
 
 
+@pytest.fixture(scope="function")
+def idea_message_column_positive_on_thread_phase(request, idea_message_column_positive, subidea_1, subidea_thread_phase_1, test_session):
+    idea_message_column_positive.idea = subidea_thread_phase_1
```

Comment:
Not sure if it will make a problem in teardown but I would rather create a new `IdeaMessageColumn` from scratch like in `idea_message_column_positive`.

Reply:
@ayyazdaniaryan suggested to me to reuse fixtures in these cases (couple other fixtures are added like this one)